bench/workgen/workgen.cpp

```diff
@@ -3034,6 +3034,8 @@ WorkloadOptions::WorkloadOptions()
     _options.add_bool("mirror_tables", mirror_tables, "mirror database operations");
     _options.add_string(
       "mirror_suffix", mirror_suffix, "the suffix to append to mirrored table names");
+    _options.add_int("background_compact", background_compact,
```

Comment:
We could also add an option of setting the timeout here.

Reply:
We could, I am just limiting the scope to enabling it for now with a specific threshold.

Reply:
Also alphabetize the list please.